express/blocks/ratings/ratings.js

```diff
@@ -124,12 +125,12 @@ export default function decorate($block) {
           value: rating,
         },
         {
-          name: 'Timestamp',
-          value: new Date().toLocaleString(),
+          name: 'Feedback',
+          value: comment,
         },
         {
-          name: 'Comment',
-          value: comment,
+          name: 'Timestamp',
+          value: new Date().toLocaleString('en-US', { timeZone: 'UTC' }),
```

Comment:
why not simply `new Date().toUTCString()`?

Reply:
We can do, but I found this to be more readable. I'll make the change!

Reply:
OK let's leave it then